app/controllers/wpcc/your_contributions_controller.rb

```diff
@@ -3,17 +3,15 @@
 module Wpcc
   class YourContributionsController < EngineController
     protect_from_forgery
+    after_action :store_eligible_salary, only: :new
```

Comment:
I'm not a huge fan of before and after filters because in my experience they can be overlooked. In this instance, its only being used by one method.